examples/asset/asset_settings.rs

```diff
@@ -0,0 +1,80 @@
+//! This example demonstrates the usage of '.meta' files and [`AssetServer::load_with_settings`] to override the default settings for loading an asset
+
+use bevy::{
+    prelude::*,
+    render::texture::{ImageLoaderSettings, ImageSampler},
+};
+
+fn main() {
+    App::new()
+        .add_plugins(
+            // This just tells the asset server to look in the right examples folder
+            DefaultPlugins.set(AssetPlugin {
+                file_path: "examples/asset/files".to_string(),
+                ..Default::default()
+            }),
+        )
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    // Without any .meta file specifying settings, the default sampler [ImagePlugin::default()] is used for loading images.
+    // If you are using a very small image and rendering it larger like seen here, the default linear filtering will result in a blurry image.
+    // Useful note: The default sampler specified by the ImagePlugin is *not* the
```

Comment:
Is not the _what_?

Reply:
Oh oops, I never noticed that I didn't finish that sentence... It was supposed to be mentioning that the image plug-in specifies a linear sampler which is not the same as the default implementation of sampler. Since this was already merged, I'll make another tiny little pr when I have a minute to fix it

Reply:
This will be addressed by #13243